docs/installation/docker.md

```diff
@@ -0,0 +1,22 @@
+# Installing Loki with Docker/Docker-compose
+
+## Prerequisites
+
+Basiclly, you must install docker.
+If you want to install loki with docker-compose, install it too.
+
+## Install with Docker
+
+```bash
+$ wget https://raw.githubusercontent.com/grafana/loki/master/cmd/loki/loki-local-config.yaml -o loki-config.yaml
+$ docker run -p 3100:3100 grafana/loki:latest -config.file=./loki-config.yaml
+$ wget https://raw.githubusercontent.com/grafana/loki/master/cmd/promtail/promtail-docker-config.yaml -o promtail-config.yaml
+$ docker run -v /var/log:/var/log grafana/promtail:latest -config.file=./promtail-config.yaml
+```
+
+## Install with Docker-compose
+
+```bash
+$ wget https://raw.githubusercontent.com/grafana/loki/master/production/docker-compose.yaml -o docker-compose.yaml
+# docker-compose -f docker-compose.yaml up
```

Comment:
Does the `#` here imply sudo? I think it should just be `$` like the other line; most people have their docker configured to not need sudo access, and those who do should know to insert it.